### DISCUSSION TITLE
Provide Debugging Capability (e.g : Debug Point between test steps )

### DISCUSSION THREAD

**Discussion by samuelvinay91** [testsigmahq/testsigma#26] [Feature Ideas]
please provide debug points in between test cases

**Comment by samuelvinay91** [comment]
I see https://github.com/testsigmahq/testsigma/issues/11 is the same as my request.

**Comment by amitsavyasachi** [comment]
@samuelvinay91 Debugger is present on the cloud version as BETA. We are considering bringing it to OS version in the next 3 Q
